mysql-test/t/mysqldump_gtid_state.test

```diff
@@ -107,6 +107,65 @@ CALL mtr.add_suppression(".*InnoDB: Table `mysql`.`innodb_table_stats` not found
 
 SHOW CREATE TABLE `mysql`.`innodb_table_stats`;
 
+
+--echo #
+--echo # Bug#105761: mysqldump make a non-consistent backup
+--echo #               with --single-transaction option
+--echo #
+
+--echo # restarting the server
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+
+# Request shutdown
+-- send_shutdown
+-- source include/wait_until_disconnected.inc
+
+--echo # Restart server.
+--exec echo "restart:--enforce-gtid-consistency=ON --gtid-mode=ON --log-bin" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+
+# Turn on reconnect
+--enable_reconnect
+
+# Call script that will poll the server waiting for it to be back online again
+--source include/wait_until_connected_again.inc
+
+TRUNCATE TABLE t;
+RESET MASTER;
+
+--write_file $MYSQL_TMP_DIR/insert_in_background.sh
+#!/bin/bash
+for i in $(seq 1 1000);
+do
+ ($MYSQL -e "use test; INSERT INTO t VALUES ($i, 1);") < /dev/null > /dev/null 2>&1
+done
+EOF
+--chmod 0755 $MYSQL_TMP_DIR/insert_in_background.sh
+--exec_in_background $MYSQL_TMP_DIR/insert_in_background.sh
```

Comment:
we can use connection with a stored procedure and use --send.  to simulate background activity?

Reply:
Done.